akka-stream-tests/src/test/scala/akka/stream/scaladsl/GraphConcatSpec.scala

```diff
@@ -103,8 +103,11 @@ class GraphConcatSpec extends TwoStreamsSetup {
       subscriber2.expectSubscriptionAndError(TestException)
     }
 
-    "work with one delayed failed and first nonempty publisher" in {
-      pending // FIXME: This relies on materialization order!!
+    "work with one nonempty and one delayed failed publisher" in {
+      // This test and the next one are materialization order dependent and rely on the fact
```

Comment:
we might break this ordering though, but I guess the comment is enough so the first one encountering the issue can modify it accordingly.